### PR TITLE
testutil/compose: add unit tests

### PR DESCRIPTION
**Pull request by corverroos**
Adds unit tests for different cluster configurations.

category: test
ticket: #609

### CODE REVIEW

File: testutil/compose/compose/smoke_internal_test.go [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/obolnetwork/charon/testutil/compose"
+)
+
+//go:generate go test . -run=TestSmoke -integration -v
+var integration = flag.Bool("integration", false, "Enable docker based integration test")
+
+func TestSmoke(t *testing.T) {
```
Comment (corverroos) [inline testutil/compose/compose/smoke_internal_test.go:32]: These are the unit tests

File: testutil/compose/static/prometheus/prometheus.yml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 global:
-  scrape_interval:     15s # Set the scrape interval to every 15 seconds. Default is every 1 minute.
-  evaluation_interval: 15s # Evaluate rules every 15 seconds. The default is every 1 minute.
+  scrape_interval:     5s # Set the scrape interval to every 15 seconds. Default is every 1 minute.
```
Comment (corverroos) [inline testutil/compose/static/prometheus/prometheus.yml:2]: scrape faster for more responsive alerts

File: testutil/compose/compose/smoke_internal_test.go (hunk 1 above)
Comment (dB2510) [reply testutil/compose/compose/smoke_internal_test.go:32]: Unit `tests` of smoke `tests` 🙃

**Review by xenowits — APPROVED** [review]
LGTM🚀

**Pull request merged**